### PR TITLE
Fixed #867 - `sbt.TrapExitSecurityException` when running ZIO's App frrom sbt

### LINKED ISSUE #867: `sbt.TrapExitSecurityException` when running ZIO's App from sbt
When I use ZIO's App and run the main class with sbt, I always get 
    `Exception: sbt.TrapExitSecurityException thrown from the UncaughtExceptionHandler in thread "run-main-x"`

Is there any way to configure my build.sbt (or sbt/jvm options) to avoid this?

### PR DESCRIPTION
**Pull request by oscarvarto**
I also did minor changes/fixes to `CONTRIBUTING.md`.

Thanks for helping me to do my first PR to ZIO.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ To clone your forked repository, first make sure you have installed [Git](https:
 ```bash
 mkdir zio
 cd zio
-git clone git@github.com:zio/zio.git .
+git clone git@github.com:your-user-name/zio.git .
```
Comment (oscarvarto) [inline CONTRIBUTING.md]: I guess contributors are expected to clone their fork.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ compile
 You can compile the test source code with the following SBT command:
 
 ```bash
-compile:test
+test:compile
```
Comment (oscarvarto) [inline CONTRIBUTING.md]: The original was wrong. This works.

File: core/jvm/src/main/scala/zio/App.scala [hunk 3]
```diff
@@ -50,7 +50,7 @@ trait App extends DefaultRuntime {
    */
   // $COVERAGE-OFF$ Bootstrap to `Unit`
   final def main(args0: Array[String]): Unit =
-    sys.exit(
+    try sys.exit(
```
Comment (oscarvarto) [inline core/jvm/src/main/scala/zio/App.scala]: The actual fix. Taken from https://github.com/zio/zio/issues/867#issuecomment-495580505.
Thanks :+1:

**Comment by jdegoes** [comment]
@oscarvarto Thank you for this contribution! I've wanted to fix this nagging pain for a long time. 

Congratulations on your first ZIO pull request!

giphy

**Pull request merged**